examples/features/torch_batch_process/batch_inference/generate_embedding/requirements.txt

```diff
@@ -0,0 +1,5 @@
+accelerate>=0.12.0
+transformers>=4.28.1,<4.29.0
```

Comment:
what is the `<4.29.0` limit for?

Reply:
I put it in to align with other [determined requirements.txt](https://github.com/search?q=repo%3Adetermined-ai%2Fdetermined%20%3C4.29.0&type=code) files that specify it.